<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CV-Drone</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="videoContainer">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Loading...</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>48</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonStart">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonEmergency">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Emergency</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDetect">
         <property name="text">
          <string>Detect</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonAutopilot">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Autopilot</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionVideo_Options"/>
    <addaction name="actionControl_Window"/>
   </widget>
   <widget class="QMenu" name="menuReconnect">
    <property name="title">
     <string>&amp;Reconnect</string>
    </property>
    <addaction name="actionReconnect"/>
    <addaction name="separator"/>
    <addaction name="actionVideo_Service"/>
    <addaction name="actionNavdata_Service"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="actionCommands"/>
    <addaction name="actionOpenCV"/>
    <addaction name="actionNavdata"/>
   </widget>
   <addaction name="menuWindow"/>
   <addaction name="menuReconnect"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionVideo_Options">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Video Settings</string>
   </property>
  </action>
  <action name="actionReconnect">
   <property name="text">
    <string>&amp;All Services</string>
   </property>
  </action>
  <action name="actionVideo_Service">
   <property name="text">
    <string>&amp;Video-Service</string>
   </property>
  </action>
  <action name="actionNavdata_Service">
   <property name="text">
    <string>&amp;Navdata-Service</string>
   </property>
  </action>
  <action name="actionCommands">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;AT*Commands</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpenCV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;OpenCV</string>
   </property>
  </action>
  <action name="actionNavdata">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Navdata</string>
   </property>
  </action>
  <action name="actionControl_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Manual Control</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonDetect</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonAutopilot</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
